nservicebus/dependency-injection/structuremap_usage_extensions.dependencyinjection_[1,).partial.md

```diff
@@ -1 +1,3 @@
 NServiceBus can be configured to use [StructureMap](https://structuremap.github.io/) for dependency injection via the [`NServiceBus.Extensions.Hosting`](/nservicebus/hosting/extensions-hosting) or the [`NServiceBus.Extensions.DependencyInjection`](/nservicebus/dependency-injection/extensions-dependencyinjection.md) package.
+
+snippet: Autofac
```

Comment:
an autofac snippet in the structuremap partial?

Reply:
Duh, fixed now